dev-docs/development_process.md

```diff
@@ -60,6 +60,8 @@ gcloud auth login
 gcloud config set project hail-vdc
 gcloud container clusters get-credentials vdc --zone=us-central1-a
 gcloud auth configure-docker
+# If you are using the Artifact Registry:
+# gcloud auth configure-docker $REGION-docker.pkg.dev
```

Comment:
Could these possibly be condensed into a phony make target that uses variables in config.mk?

Reply:
Something like this?
https://github.com/hail-is/hail/pull/10107/commits/a3c892f21bf84a16ae72f99c40dac17445a650de

Though it required to add `include ../config.mk` into `hail/Makefile` - not sure if this will break any assumption?

Also not sure if it makes sense to parameterize the cluster name (`vdc`). We didn't override it for us, but curious about the general case.